cms/core/blocks/panels.py

```diff
@@ -3,27 +3,34 @@
 from wagtail import blocks
 
 
-class PanelBlock(blocks.StructBlock):
-    """DS Panel block.
-    https://service-manual.ons.gov.uk/design-system/components/panel
-    https://service-manual.ons.gov.uk/design-system/components/announcement-panel
-    https://service-manual.ons.gov.uk/design-system/components/success-panel
-    https://service-manual.ons.gov.uk/design-system/components/warning-panels.
+class BasePanelBlock(blocks.StructBlock):
+    """A reusable base panel block with a body field.
+    Subclasses can override Meta attributes (e.g., template, label)
+    and define additional fields as needed.
     """
 
-    variant = blocks.ChoiceBlock(
-        choices=[
-            ("warn", _("Warning")),
-            ("info", _("Information")),
-            ("announcement", "Announcement"),
-            ("error", "Error"),
-            ("success", "Success"),
-        ],
-        default="warn",
-    )
     body = blocks.RichTextBlock(features=settings.RICH_TEXT_BASIC)
-    title = blocks.CharBlock(required=False, label=_("Title (optional)"))
 
     class Meta:
-        label = _("Warning or information panel")
-        template = "templates/components/streamfield/panel_block.html"
+        abstract = True
+        group = _("Panels")
+
+
+class WarningPanelBlock(BasePanelBlock):
+    class Meta:
+        template = "templates/components/streamfield/warning_panel.html"
+        label = _("Warning Panel")
+
+
+class AnnouncementPanelBlock(BasePanelBlock):
+    class Meta:
+        template = "templates/components/streamfield/announcement_panel.html"
+        label = _("Announcement Panel")
```

Comment:
```suggestion
        icon = "pick"
        label = _("Announcement Panel")
```

Let's get this in like this. 
A follow could be to add https://fontawesome.com/icons/bullhorn?f=classic&s=solid to our icons and register it with the others in `core.wagtail_hooks.register_icons`

Reply:
Noted!

Reply:
https://jira.ons.gov.uk/browse/CMS-345